arduino-ide-extension/src/browser/dialogs/settings/settings-step-input.tsx

```diff
@@ -86,34 +52,14 @@ const SettingsStepInput: React.FC<SettingsStepInputProps> = (
     const number = Number(eventValue);
 
     if (!isNaN(number) && number !== value) {
-      let newValue;
```

Comment:
I assume the code could return with the function above 👆

```ts
if (eventValue === '') {
  setSettingsStateValue(0);
  return; // <---  this?
}
```

Reply:
yes! sorry I saw this after the fact, I'll add it in when we tackle the scale range question.